styles.scss

```
@@ -1,40 +1,416 @@
-@import "../scss/variables/general.scss";
-@import "../scss/mixins/general.scss";
+@import "./scss/variables/general.scss";
+@import "./scss/mixins/general.scss";
+@font-face {
+    font-family: "SenBold";
+    src:   url("./public/fonts/Sen-Bold.ttf") format("truetype");
+    font-weight: bold;
+    font-style: normal;
+}
+@font-face {
+    font-family: "SenRegular";
+    src: url("./public/fonts/Sen-Regular.ttf") format("truetype");
+    font-weight: normal;
+    font-style: normal;
+    font-display: swap;
+}
+@font-face {
+    font-family: "Inter";
+    src: url("./public/fonts/Inter-Regular.ttf") format("truetype"); 
+    font-weight: normal;
+    font-style: normal;
+    font-display: swap;
+}
 
 body {
     margin: 0;
     padding: 0;
 }
-// nested scss
-.header {
+// reset css
+*{
+    padding: 0; margin:0;
+    box-sizing: border-box; 
+    outline: none; border: none;
+    text-decoration: none;
+  
+}
+html{
+    font-size: 62.5%;
+    overflow-x: hidden;
+    scroll-padding-top: 9rem;
+    scroll-behavior: smooth;
+    
+}
+/* độ rộng */
+::-webkit-scrollbar{
+    width: .8rem;
+}
+/* cái nút kéo */
+::-webkit-scrollbar-thumb{
+   background: linear-gradient(transparent,#30ff00);
+   border-radius: 5rem;
+}
+::-webkit-scrollbar-thumb:hover{
+    background: linear-gradient(transparent,#00c6ff);
+}
+/* cái thanh dài */
+::-webkit-scrollbar-track{
+    width: .8rem;
+}
+.header{
+    width: 100%;
+    @include displayFlexCenter(center,space-between,32px);
+    padding: 1.2rem 8rem 1.2rem 8.2rem;
+    background-color: $darkBg;
+    &__logo{
+        font-size: 24px;
+        font-weight: 700px;
+        color: $white;
+    }
+    &__subcribe{
+        padding: 16px 48px;
+        @include fonts(SenBold,1.8rem,700,2.4rem);
+        color: $darkBg;
+        background-color: $white;
+        cursor: pointer;
+    }
+    &__subcribe:hover{
+        opacity: 0.7;
+    }
+    &__links{
+     flex-grow: 1;
+     display: flex;
+     @include displayFlexCenter(center,flex-end,3.2rem)
+
+    }
+    &__links a{
+        font-size: 1.6rem;
+        font-weight: 400;
+        line-height: 2.8rem;
+        color: $white;
+    }
+        &__links a:hover{
+            opacity: 0.7;
+        }
+
+}
+.post-featured{
     width: 100%;
-    height: 80px;
-    padding: 0px 80px 0px 83px;
     box-sizing: border-box;
+    padding: 8rem;
+
+    @include displayFlexCenter(flex-start,space-between);
+    gap: 14.1rem;
+
+    background-color: $lavendar;
+}
+.post-featured__content{
```

tại sao không viết nested kiểu scss?
```suggestion
post-featured__content{
&__content{
```